src/odemis/acq/align/test/delphi_test.py

```diff
@@ -104,7 +104,13 @@ def test_find_hole_center(self):
         data[0].shape = Y, X
 
         hole_coordinates = delphi.FindCircleCenter(data[0], 0.02032, 6, darkest=True)
-        expected_coordinates = (0.0052705, -0.0018415)  # (391.5, 257.5) px
+
+        # depending on the cv2 version which is related to the Ubuntu version the expected outcome is different
+        if ".".join(delphi.cv2.__version__.split(".")[:2]) == "3.2":
+            expected_coordinates = (0.0052705, -0.0018415)  # (391.5, 257.5) px
+        else:
+            expected_coordinates = (0.0046355, -0.0028575)  # (386.5, 265.5) px
```

Comment:
If  both results are correct, because we don't need to be very precise, then I would suggest to explicitly check that the returned coordinates are "not too far" from the expected center. Hardcoding the values to 2 very precise tuples is confusing. Please have one "expected coordinates" and check the distance of the measured coordinates is less than N meters (where you have to pick N a small constant).